pom.xml

```diff
@@ -62,7 +62,7 @@ limitations under the License.
     </distributionManagement>
 
     <properties>
-        <revision>3.0-SNAPSHOT</revision>
+        <revision>3.2-SNAPSHOT</revision>
```

Comment:
Uh? should we also fix this in release-3.1 branch?

Reply:
I did and pushed directly on release-3.1 just now. We never noticed this before...